data/mods/Magiclysm/monsters/monsters.json

```
@@ -114,6 +114,83 @@
       }
     ]
   },
+  {
+    "id": "mon_hunllef_lamb",
+    "type": "MONSTER",
+    "name": { "str": "lamb" },
+    "description": "The offspring of the Hunllef rams are much more docile but still have wicked looking horns like their parents.",
+    "default_faction": "hunllef",
+    "bodytype": "dog",
+    "categories": [ "WILDLIFE" ],
+    "species": [ "MAMMAL" ],
+    "volume": "50750 ml",
+    "weight": "50750 g",
+    "hp": 38,
+    "speed": 90,
+    "material": [ "flesh" ],
+    "symbol": "s",
+    "color": "dark_gray",
+    "looks_like": "mon_sheep",
+    "melee_skill": 2,
+    "melee_dice": 1,
+    "melee_dice_sides": 4,
+    "melee_cut": 1,
+    "dodge": 2,
+    "anger_triggers": [  ],
+    "harvest": "mammal_small_wool",
+    "fear_triggers": [ "SOUND", "PLAYER_CLOSE", "FRIEND_ATTACKED" ],
+    "placate_triggers": [ "PLAYER_WEAK" ],
+    "death_function": [ "NORMAL" ],
+    "upgrades": { "age_grow": 240, "into": "mon_sheep" },
+    "//": "Puberty reached in 6-9 months.",
+    "biosignature": { "biosig_item": "feces_manure", "biosig_timer": 7 },
+    "flags": [ "SEES", "HEARS", "SMELLS", "ANIMAL", "PATH_AVOID_DANGER_1", "WARM" ]
+  },
+  {
+    "id": "mon_nightmare_sheep",
+    "type": "MONSTER",
+    "name": { "str_sp": "hunllef rams" },
+    "description": "Hunlef rams, also known as Nightmare Sheep are a particularly vicious breed of sheep suspected to be from another plane of existence.  Both males and females display deadly horns and a willingness to use them on most other creatures that come near them.  They are the most biddable source of parabolan wool and often take a protective stance towards other sheep, so some farmers will risk life and limb to keep them.",
+    "default_faction": "hunllef",
+    "bodytype": "dog",
+    "categories": [ "WILDLIFE" ],
+    "species": [ "MAMMAL" ],
+    "volume": "111500 ml",
+    "weight": "111500 g",
+    "hp": 130,
+    "speed": 90,
+    "material": [ "flesh" ],
+    "symbol": "s",
+    "color": "dark_gray",
+    "melee_skill": 4,
+    "melee_dice": 3,
+    "melee_dice_sides": 6,
+    "melee_cut": 2,
+    "dodge": 2,
+    "starting_ammo": { "milk_raw": 5 },
+    "anger_triggers": [ "HURT", "PLAYER_CLOSE", "PLAYER_WEAK", "FRIEND_ATTACKED" ],
```

```suggestion
    "anger_triggers": [ "HURT", "PLAYER_CLOSE", "PLAYER_WEAK", "FRIEND_ATTACKED", "PLAYER_NEAR_BABY" ],
```
PLAYER_CLOSE in both anger and fear triggers? And do you have in mind something other than 0 for aggression and/or morale?